src/pages/workspace/companyCards/assignCard/BankConnection/index.tsx

```diff
@@ -31,9 +32,10 @@ function BankConnection({policyID, feed}: BankConnectionStepProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
     const bankName = getBankName(feed);
-    const currentUrl = getCurrentUrl();
-    const isBankConnectionCompleteRoute = currentUrl.includes(ROUTES.BANK_CONNECTION_COMPLETE);
```

Comment:
@VickyStash Hmm I am not sure about this change - we basically go back to what we had before, relying on the pusher update to the `SHARED_NVP_PRIVATE_DOMAIN_MEMBER` but Vicky changed it to use the url - when we complete the connection we should to to the bank-connection-completed route. Why is that not working in this case?